packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -20,6 +20,79 @@ Markup: table-complex.example.html
 Style guide: components.table.complex
 */
 
+/*
+`<Table>`
+
+`Table` and sub-table components render data tables using semantic markup and ARIA attributes.
```

Comment:
Maybe we can tighten up this description, similar to `Tabs`

`Table` is a container component that contains `TableCaption`, `TableHead` and `TableBody` as children, as well as `TableRow`, `TableHeaderCell` and `TableDataCell` for the table content. While the usage of these components follows ordinary HTML table semantics, this React implementation includes additional responsive features including horizontal scrolling and vertically stacked rows.